utils.ts

```diff
@@ -1,3 +1,5 @@
 export function test () {
   return null
 }
+
```

Comment:
Thoughts on doing something like this?

```suggestion
+ const new Test()
- const new Server()
```